app/views/users/profiles/edit.html.haml

```diff
@@ -0,0 +1,21 @@
+%h1= t('edit') + t('.profile', name: @profile.user.name)
```

Comment:
If you need  to have string "Edit Anastasia Balaban Profile", you should have in locale `Edit %{name} Profile`.
don't use string concatenation